spec/coverage.rb

```diff
@@ -6,5 +6,5 @@
   formatters << Coveralls::SimpleCov::Formatter
 end
 
-SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter[*formatters]
+SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter.new(formatters)
```

Comment:
This change does not work with coveralls v0.8.2, the result is a "`initialize': wrong number of arguments (1 for 0) (ArgumentError)". I happen to have coveralls v0.8.2 installed as it both fulfils the cucmber-ruby(-core) requirement of `'~>0.7'` and the gherkin requirement of `'~> 0.8', '< 0.8.8'`.